deployment/clowdapp.yml

```diff
@@ -18,6 +18,8 @@ objects:
     - compliance-ssg
     - config-manager
     - cloud-connector
+    # add MQTT mock (mosquitto) from cloud-connector
+    - mosquitto
```

Comment:
We need to check whether this wouldn't cause issues on stage/prod. Event cloud-connector does not lists this as an (optional) dependency, as it is used only for testing on ephemeral envs.

We can follow the same, but then the cloud-connector needs to be deployed explicitly, as `mosquitto` is pulled out of possible components of the "cloud-connector" app (as per app-interface definitions).

Note, that these lists components and not app names.

Reply:
Yes - let's not break prod, please 😬...